common/src/main/java/io/pravega/common/cluster/ClusterException.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.common.cluster;
+
+public class ClusterException extends RuntimeException {
```

Comment:
I'm not sure this should be runtime. Do we expect that there is nothing the caller can do about this?